lib/http.js

```diff
@@ -417,7 +447,38 @@ async function _handleOperation({req, expectedRootCapability, keystoreId}) {
   return runOperation({operation});
 }
 
+function _verifyRequestIp({keystoreConfig, req}) {
+  const {ipAllowList} = keystoreConfig;
+  if(!ipAllowList) {
+    return {verified: true};
+  }
+
+  // the first IP in the sourceAddresses array will *always* be the IP
+  // reported by Express.js via `req.connection.remoteAddress`. Any additional
+  // IPs will be from the `x-forwarded-for` header.
+  const sourceAddresses = forwarded(req);
+
+  let verified = false;
+  // ipAllowList is an array of CIDRs
+  for(const cidr of ipAllowList) {
+    const netmask = new Netmask(cidr);
+    for(const address of sourceAddresses) {
+      if(netmask.contains(address)) {
+        verified = true;
```

Comment:
We should just return here -- no need to keep iterating through the `ipAllowList`.